src/parser/mage/frost/modules/checklist/Component.js

```diff
@@ -46,18 +47,38 @@ class FrostMageChecklist extends React.PureComponent {
           {combatant.hasTalent(SPELLS.RAY_OF_FROST_TALENT.id) && <AbilityRequirement spell={SPELLS.RAY_OF_FROST_TALENT.id} />}
           {combatant.hasTalent(SPELLS.ICE_NOVA_TALENT.id) && <AbilityRequirement spell={SPELLS.ICE_NOVA_TALENT.id} />}
         </Rule>
+        <Rule
+          name="Shatter your spells"
+          description={(
+            <>
+              The biggest aspect to etting the most damage out of your rotation, is to ensure that you are <SpellLink id={SPELLS.SHATTER.id} />ing as many of your spells as possible while they are frozen or acting as if they are frozen. The biggest aspect of this is taking advantage of the <SpellLink id={SPELLS.WINTERS_CHILL.id} /> debuff. Winter's Chill is applied to the target when you use a <SpellLink id={SPELLS.BRAIN_FREEZE.id} /> proc and makes the target act as if it is frozen for a short duration of time. Therefore, you should cast a rotational ability like <SpellLink id={SPELLS.FROSTBOLT.id} />, <SpellLink id={SPELLS.EBONBOLT_TALENT.id} />, or <SpellLink id={SPELLS.GLACIAL_SPIKE_TALENT.id} />, followed immediately by the Brain Freeze buffed Flurry and then end with an <SpellLink id={SPELLS.ICE_LANCE.id} />. You can also do other things like use your pet's <SpellLink id={SPELLS.FREEZE.id} /> or other abilities that freeze non boss enemies to shatter spells as well. 
```

Comment:
A few typos, and some extra words; my recommendation:

`The most important element of maximizing the damage of your rotation is ensuring that you are <SpellLink id={SPELLS.SHATTER.id} />ing as many of your spells as possible. The key aspect of this is taking advantage of the <SpellLink id={SPELLS.WINTERS_CHILL.id} /> debuff. Winter's Chill is applied to the target when you use a <SpellLink id={SPELLS.BRAIN_FREEZE.id} /> proc and makes the target act as if it is frozen for a short duration of time. Therefore, you should cast a rotational ability like <SpellLink id={SPELLS.FROSTBOLT.id} />, <SpellLink id={SPELLS.EBONBOLT_TALENT.id} />, or <SpellLink id={SPELLS.GLACIAL_SPIKE_TALENT.id} />, followed immediately by the Brain Freeze buffed Flurry and then end with an <SpellLink id={SPELLS.ICE_LANCE.id} />. Against non-boss enemies, you can also utilize other things like <SpellLink id={SPELLS.FROST_NOVA.id} /> or your pet's <SpellLink id={SPELLS.FREEZE.id} /> to shatter spells as well.`